pkg/typeutil/conversion.go

```diff
@@ -68,3 +68,20 @@ func JSONToUint64Slice(from interface{}) ([]uint64, bool) {
 	}
 	return to, true
 }
+
+// JSONToStringSlice converts JSON slice to string slice.
+func JSONToStringSlice(from interface{}) ([]string, bool) {
```

Comment:
Can we use `reflect` to merge this function with `JSONToUint64Slice`?

Reply:
Maybe not? It seems we only need  to support string and uint64, while using `reflect` costs more and also harder to maintain?